plat/kvm/arm/link64.lds.S

```diff
@@ -73,6 +73,7 @@ SECTIONS {
 	_text = .;
 	.text :
 	{
+		KEEP (*(.data.boot))
```

Comment:
Sorry, maybe I'm missing something but where do you use this in `arm64`?

Reply:
Leftover from when I had `pe_header.S` and that was an assembly written source for this specific section, so that it transparently works for both arch's. It's fine to remove now. Nice catch!